packages/core/src/types.ts

```diff
@@ -1045,26 +1052,38 @@ export declare namespace SCXML {
   }
 }
 
+export type Spawnable =
+  | MachineNode<any, any, any>
+  | PromiseLike<any>
+  | InvokeCallback
+  | Subscribable<any>;
+
 // Taken from RxJS
 export interface Unsubscribable {
   unsubscribe(): any | void;
 }
+
+export interface Observer<T> {
+  // Sends the next value in the sequence
+  next?: (value: T) => void;
+
+  // Sends the sequence error
+  error?: (errorValue: any) => void;
+
+  // Sends the completion notification
+  complete: () => void;
+}
+
 export interface Subscribable<T> {
+  subscribe(observer: Observer<T>): Unsubscribable | undefined;
```

Comment:
both of those subscribes define different things as required and optional, in the first one only `complete` is required, whereas in the second one only `next` is required, I believe both should be aligned to be the same